objects/adffc0/contained/706d9f/contained/4363a8/script.lua

```diff
@@ -0,0 +1,299 @@
+local balancedBag = "ab7fce"
+local thematicBag = "bb7fce"
```

Comment:
looks like this bag is missing from review

Reply:
Indeed. We don't have drowned land tiles for the thematic boards yet, so it'll be an empty bag, but I can add it in.